egs2/yesno/asr1/local/data.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $@"
+}
+SECONDS=0
+
+stage=-1
+stop_stage=1
+
+datadir=./downloads
```

Comment:
Sorry for again, I changed my mind, so could you also use ${YESNO} here and write it db.sh?